### DISCUSSION TITLE
New G6 marked as FAILED even before start

### DISCUSSION THREAD

**Discussion by jw243** [NightscoutFoundation/xDrip#2319] [Q&A]
I am in the process of testing Dexcom G6 as part of the "Hello Dexcom" scheme. With my first sensor, I got a **Sensor marked as FAILED** error message as described [here](https://github.com/NightscoutFoundation/xDrip/discussions/2318).
Dexcom was so kind as to send me a new sensor. I connected it, just like the first sensor, with XDrip following [this guideline](https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html), but didn't start it.
I got [proper connection](https://navid200.github.io/xDrip/docs/Proper-connectivity.html), but immediately the **sensor status** was marked **Sensor Failed 5**, just like with the first sensor. Hence, I didn't dare to start the sensor in XDrip.
Instead, I switched Hardware Data Source to "Disable Collection", installed the Dexcom app and started the sensor with it. The sensor worked fine and collected data after the warmup period of 2 hours.
Now I would be interested to know:

- Was the sensor failure warning just a remnant of the first (failed) sensor?
- Would I have been able to start the sensor anyway despite the sensor failure warning and everything would have worked as normal?
- Or would the sensor indeed have failed with XDrip?
- If the last thing is correct, why would all of my sensors fail with XDrip despite me meticulously following the setup procedure?

Thanks a lot for your input.

**Comment by Navid200** [comment]
[This]{https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html} is not a sensor start guideline.  It lists the recommended settings as well as the Android settings required.

These are the guidelines for starting a sensor:  https://navid200.github.io/xDrip/docs/Starting-G6.html

I would appreciate it if you ask us for help instead of contacting Dexcom. 
It would only be fair to contact Dexcom if you use their recommended devices and apps.  Dexcom does not recommend that you use xDrip.
xDrip is a sophisticated tool that can be used for a plethora of devices.  It is complicated and if you don't use it right, you will have issues.  But, we will help you if you give us a chance, instead of contacting Dexcom.

There are times a tag remains on the system status page for a few read cycles before clearing.  This could happen to the sensor fail tag as well.
Almost everything on the system status page (G5/G6 status page) is a representation of what xDrip receives from the transmitter.  So, if you tap on start, what's shown on the system status page may take 5 minutes to be updated to what is inside the transmitter.

There is no way to have a continuous connection to the transmitter.  The battery would not last 3 months then.

**Reply by jw243** [reply]
Thanks for getting back to me so quickly.

> I would appreciate it if you ask us for help instead of contacting Dexcom.
It would only be fair to contact Dexcom if you use their recommended devices and apps. Dexcom does not recommend that you use xDrip.

Of course, I didn't contact Dexcom about my problems with xDrip. I contacted them because I couldn't get the sensor to work at all, even with the Dexcom app. They sent me a replacement, because the sensor indeed seemed to be faulty.

> There are times a tag remains on the system status page for a few read cycles before clearing. This could happen to the sensor fail tag as well.

So, does this mean I could have tried to start the sensor in xDrip **despite** the "sensor marked as failed" message? I was afraid I could damage the sensor if I did, that's why I didn't try it. Instead I started it with the Dexcom app and it worked without problems (and the sensor was abviously not faulty).

Thanks again for your help.

**Reply by Navid200** [reply]
You are welcome to use xDrip.
If you do, it is only fair to follow our instructions.

I can show you how easy it is to cause a failure with a new sensor.  After that, if you attempt to start it with another app, like the Dexcom app, the transmitter will not allow it because it detects an attempt to restart a sensor.
If you contact Dexcom, they assume that something is wrong with the sensor.

If you give me a chance to check everything and I tell you that something seems to be wrong with the sensor, then, you can be certain that something is indeed wrong with the sensor.

Without going through a few tests, I have no way of knowing for sure what happened to your sensor.  It could indeed have been a faulty sensor.  Although, that is extremely unlikely.  Possible, but, unlikely.  Dexcom production control seems to high quality.  I have no reason to expect faulty products to be in circulation.  But, it's possible.
